Documentation/operation docs rework (#3074)
scality/metalk8s: docs/operation/cluster_monitoring/index.rst

Cluster Monitoring
==================

> This chapter covers the MetalK8s monitoring and alerting stack operations. It

Reviewer: ```suggestion
This section covers the MetalK8s monitoring and alerting stack operations. It
```

also describes the metric resources that are automatically monitored once

> MetalK8s is deployed (i.e: Prometheus), and the list of alerting and recording

Reviewer: i.e. Prometheus (period, no colon). However, this looks like an e.g. to me. 

Is it Prometheus only? If so: _i.e._ 
If it's Prometheus (as one of several possibles) then: _e.g._

Better: just say what's meant, using the words "for example" or "that is".

> This section covers the MetalK8s monitoring and alerting stack operations. It
> also describes the metric resources that are automatically monitored once
> MetalK8s is deployed (for example Prometheus), and the list of pre-configured
> alerting and recording rules.

Reviewer 2: Unsure what "metric resources" mean, and why Prometheus is an example of that.

I'd suggest:

```suggestion
This section covers the MetalK8s monitoring and alerting stack operations. It
also describes the metrics monitored using Prometheus, with the list of
pre-configured alerting and recording rules.
```

> rules which are pre-configured.

Reviewer: rules that are pre-configured (i.e., the list of pre-configured alerting and recording rules.)
or 
rules, which is configured (i.e., the pre-configured list of alerting and recording rules.) 

The best to my ear if I read this correctly: 
the list of pre-configured alerting and recording rules.
